spec/template.dd

```diff
@@ -53,7 +53,14 @@ $(GNAME TemplateParameter):
         $(I TemplateParameter) in case one is not supplied.
     )
 
-$(H2 $(LNAME2 explicit_tmp_instantiation, Explicit Template Instantiation))
+    $(NOTE `template` keyword declarations are less common in idiomatic D code.
```

Comment:
Is this note necessary? Also it should probably be "less common than [something]" if this point is to be made.

Reply:
The 'than' is on the next line. The point is for people reading the page as a learning resource rather than a reference (the page seems intended for that as it gradually introduces things). They should not get the idea that verbose template blocks with explicit instantiation is typical in D.

Reply:
@maxhaton I've reworded this a bit to mention single member templates, see 640802e